### ISSUE TITLE
[Bug] Table not displaying problem

### ISSUE THREAD

**Issue by smartyhero** [AppFlowy-IO/AppFlowy#6104]
### Bug Description

<img width="1437" alt="image" src="https://github.com/user-attachments/assets/db31f1ed-9ee5-4a23-a443-1b2a0f7e50ba">

logs:
```
{"msg":"[REFRESH_USER_PROFILE - END]","time":"08-29 18:13:33","target":"flowy_user::user_manager::manager"}
flutter: Bad state: No element
flutter: #0      Iterable.first (dart:core/iterable.dart:643)
flutter: #1      new TableNode (package:appflowy_editor/src/editor/block_component/table_block_component/table_node.dart:52)
flutter: #2      TableBlockComponentBuilder.build (package:appflowy_editor/src/editor/block_component/table_block_component/table_block_component.dart:99)
flutter: #3      BlockComponentRenderer.build.<anonymous closure> (package:appflowy_editor/src/editor/editor_component/service/renderer/block_component_service.dart:143)
flutter: #4      BlockComponentContainerState.build.<anonymous closure> (package:appflowy_editor/src/editor/editor_component/service/renderer/block_component_container.dart:38)
flutter: #5      Consumer.buildWithChild (package:provider/src/consumer.dart:179)
flutter: #6      SingleChildStatelessWidget.build (package:nested/nested.dart:259)
flutter: #7      StatelessElement.build (package:flutter/src/widgets/framework.dart:5557)
flutter: #8      SingleChildStatelessElement.build (package:nested/nested.dart:279)
flutter: #9      ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5487)
flutter: #10     Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
flutter: #11     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:5469)
flutter: #12     ComponentElement.mount (package:flutter/src/widgets/framework.dart:5463)
flutter: #13     SingleChildWidgetElementMixin.mount (package:nested/nested.dart:222)
flutter: #14     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4340)
flutter: #15     Element.updateChild (package:flutter/src/widgets/framework.dart:3849)
flutter: #16     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
flutter: #17     Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
flutter: #18     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:5469)
flutter: #19     ComponentElement.mount (package:flutter/src/widgets/framework.dart:5463)
flutter: #20     _InheritedProviderScopeElement.mount (package:provider/src/inherited_provider.dart:411)
flutter: #21     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4340)
flutter: #22     Element.updateChild (package:flutter/src/widgets/framework.dart:3849)
flutter: #23     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
flutter: #24     Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
flutter: #25     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:5469)
flutter: #26     ComponentElement.mount (package:flutter/src/widgets/framework.dart:5463)
flutter: #27     SingleChildWidgetElementMixin.mount (package:nested/nested.dart:222)
flutter: #28     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4340)
flutter: #29     Element.updateChild (package:flutter/src/widgets/framework.dart:3849)
flutter: #30     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
flutter: #31     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5650)
flutter: #32     Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
flutter: #33     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:5469)
flutter: #34     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:5641)
flutter: #35     ComponentElement.mount (package:flutter/src/widgets/framework.dart:5463)
flutter: #36     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4340)
flutter: #37     Element.updateChild (package:flutter/src/widgets/framework.dart:3849)
flutter: #38     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:6769)
flutter: #39     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4340)
flutter: #40     Element.updateChild (package:flutter/src/widgets/framework.dart:3849)
flutter: #41     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:6769)
flutter: #42     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4340)
flutter: #43     Element.updateChild (package:flutter/src/widgets/framework.dart:3849)
flutter: #44     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
flutter: #45     Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
flutter: #46     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:5469)
flutter: #47     ComponentElement.mount (package:flutter/src/widgets/framework.dart:5463)
flutter: #48     _RegisteredElement.mount (package:appflowy_editor/src/flutter/scrollable_positioned_list/src/element_registry.dart:68)
flutter: #49     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4340)
flutter: #50     Element.updateChild (package:flutter/src/widgets/framework.dart:3849)
flutter: #51     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:6769)
flutter: #52     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4340)
flutter: #53     Element.updateChild (package:flutter/src/widgets/framework.dart:3849)
flutter: #54     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
flutter: #55     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5650)
flutter: #56     Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
flutter: #57     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:5469)
flutter: #58     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:5641)
flutter: #59     ComponentElement.mount (package:flutter/src/widgets/framework.dart:5463)
flutter: #60     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4340)
flutter: #61     Element.updateChild (package:flutter/src/widgets/framework.dart:3849)
flutter: #62     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
flutter: #63     Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
flutter: #64     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:5469)
flutter: #65     ComponentElement.mount (package:flutter/src/widgets/framework.dart:5463)
flutter: #66     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4340)
flutter: #67     Element.updateChild (package:flutter/src/widgets/framework.dart:3849)
flutter: #68     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
flutter: #69     Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
flutter: #70     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:5469)
flutter: #71     ComponentElement.mount (package:flutter/src/widgets/framework.dart:5463)
flutter: #72     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4340)
flutter: #73     Element.updateChild (package:flutter/src/widgets/framework.dart:3849)
flutter: #74     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
flutter: #75     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5650)
flutter: #76     Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
flutter: #77     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:5469)
flutter: #78     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:5641)
flutter: #79     ComponentElement.mount (package:flutter/src/widgets/framework.dart:5463)
flutter: #80     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4340)
flutter: #81     Element.updateChild (package:flutter/src/widgets/framework.dart:3849)
flutter: #82     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
flutter: #83     Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
flutter: #84     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:5469)
flutter: #85     ComponentElement.mount (package:flutter/src/widgets/framework.dart:5463)
flutter: #86     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4340)
flutter: #87     Element.updateChild (package:flutter/src/widgets/framework.dart:3849)
flutter: #88     SliverMultiBoxAdaptorElement.updateChild (package:flutter/src/widgets/sliver.dart:860)
flutter: #89     SliverMultiBoxAdaptorElement.createChild.<anonymous closure> (package:flutter/src/widgets/sliver.dart:845)
flutter: #90     BuildOwner.buildScope (package:flutter/src/widgets/framework.dart:2845)
flutter: #91     SliverMultiBoxAdaptorElement.createChild (package:flutter/src/widgets/sliver.dart:837)
flutter: #92     RenderSliverMultiBoxAdaptor._createOrObtainChild.<anonymous closure> (package:flutter/src/rendering/sliver_multi_box_adaptor.dart:360)
flutter: #93     RenderObject.invokeLayoutCallback.<anonymous closure> (package:flutter/src/rendering/object.dart:2688)
flutter: #94     PipelineOwner._enableMutationsToDirtySubtrees (package:flutter/src/rendering/object.dart:1097)
flutter: #95     RenderObject.invokeLayoutCallback (package:flutter/src/rendering/object.dart:2688)
flutter: #96     RenderSliverMultiBoxAdaptor._createOrObtainChild (package:flutter/src/rendering/sliver_multi_box_adaptor.dart:349)
flutter: #97     RenderSliverMultiBoxAdaptor.insertAndLayoutChild (package:flutter/src/rendering/sliver_multi_box_adaptor.dart:497)
flutter: #98     RenderSliverList.performLayout.advance (package:flutter/src/rendering/sliver_list.dart:239)
flutter: #99     RenderSliverList.performLayout (package:flutter/src/rendering/sliver_list.dart:281)
  2024-08-29 18:13:34  INFO client_api::notify: old token:GotrueTokenResponse { expires_at: 1724881956, token_type: "bearer" }, new token:GotrueTokenResponse { expires_at: 1724998431, token_type: "bearer" }
    at /Users/runner/.cargo/git/checkouts/appflowy-cloud-875aed6322f3953d/7878a01/libs/client-api/src/notify.rs:55
```

### How to Reproduce

I don't know how to reproduce this. I opened appflowy today and it became like this.

### Expected Behavior

The table can be displayed normally

### Operating System

mac

### AppFlowy Version(s)

0.6.7

**Comment by theimaginationless** [comment]
I've got that with cmd+z (undo) changes with table editing
https://github.com/AppFlowy-IO/AppFlowy/issues/6146

**Comment by smartyhero** [comment]
![image](https://github.com/user-attachments/assets/3f4fbff5-7f3c-4dc8-92bd-f376bf52419c)

**Comment by smartyhero** [comment]
After I upgraded to 0.6.8, my table disappeared completely

**Comment by smartyhero** [comment]
Now the tables in my document seem to have become like this. It seems that every time I upgrade the version, the table will become like this.
![image](https://github.com/user-attachments/assets/d5bf6e15-738a-4805-893a-0addebb4e8c0)

**Comment by annieappflowy** [comment]
We have refactored the code for Simple Table. The new simple table will be released in the upcoming version (v0.7.8). The listed bug in this issue should be resolved.
